dapps/marketplace/src/components/Attestations.js

```diff
@@ -1,19 +1,46 @@
 import React from 'react'
+import { mapVerifiedAttestations } from 'utils/profileTools'
 
-const Attestations = ({ profile = {} }) => (
-  <div className="attestations">
-    {profile.emailVerified && <div className="attestation email" />}
-    {profile.phoneVerified && <div className="attestation phone" />}
-    {profile.facebookVerified && <div className="attestation facebook" />}
-    {profile.twitterVerified && <div className="attestation twitter" />}
-    {profile.airbnbVerified && <div className="attestation airbnb" />}
-    {profile.googleVerified && <div className="attestation google" />}
-    {profile.websiteVerified && <div className="attestation website" />}
-    {profile.kakaoVerified && <div className="attestation kakao" />}
-    {profile.githubVerified && <div className="attestation github" />}
-    {profile.linkedinVerified && <div className="attestation linkedin" />}
-    {profile.wechatVerified && <div className="attestation wechat" />}
-  </div>
-)
+const Attestations = ({ profile = {} }) => {
+  const verifiedAttestations = mapVerifiedAttestations(profile)
```

Comment:
Can we just loop over the array returned from GraphQL?

Reply:
I thought about that. But we will end up with lots of `.find`s or for loops. We will be iterating the same array again and again.

Reply:
If GraphQL returns them in the correct order, could we say `verfiedAttestations.map(a => <div className={'attestation ${a.id}'} />`

Reply:
Those are returned in the order the user added them. But can sort them if you want.

Reply:
Yea I think sorting them to keep everything consistent at the GraphQL level makes sense... will cut down on duplication code in the dapp too

Reply:
Sure. 👍